libs/api/domains/financial-aid/src/lib/financialAid.module.ts

```diff
@@ -0,0 +1,36 @@
+import { Module, DynamicModule } from '@nestjs/common'
+import { FinancialAidResolver } from './financialAid.resolver'
+import { FinancialAidService } from './financialAid.service'
+import { ApplicationApi, Configuration } from '../../gen/fetch'
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+
+export interface Config {
+  baseApiUrl: string
+}
+
+@Module({})
+export class FinancialAidModule {
+  static register(config: Config): DynamicModule {
+    return {
+      module: FinancialAidModule,
+      providers: [
+        FinancialAidResolver,
+        FinancialAidService,
+        {
+          provide: ApplicationApi,
+          useFactory: async () =>
+            new ApplicationApi(
+              new Configuration({
+                fetchApi: createEnhancedFetch({
+                  name: 'FinancialAidModule.applicationApi',
+                  timeout: 60000,
```

Comment:
Why the long timeout? Is the API slow? :)

Reply:
No not at all, will remove, this is just dev c/p and forgot to remove 🙈